### PR TITLE
Add a recipe for tempest-extremes

### PR DESCRIPTION
**Pull request by xylar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble be please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/tempest-extremes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    framework, we can study the joint characteristics of extremes while
+    minimizing the total data burden.
+  doc_url: https://climate.ucdavis.edu/tempestextremes.php
+  dev_url: https://github.com/ClimateGlobalChange/tempestextremes
+
+extra:
+  recipe-maintainers:
+    - xylar
+    - chengzhuzhang
+    - forsyth2
```
Comment (xylar) [inline recipes/tempest-extremes/meta.yaml:99]: @chengzhuzhang and @forsyth2, would you please confirm that you are willing to be maintainers of this feedstock?

**Comment by xylar** [comment]
@conda-forge/help-c-cpp, I am still waiting for confirmation from my colleagues if they are willing to be maintainers but this PR is otherwise ready to review.  Please let me know if you have concerns or suggestions.  A few notes that might be relevant:
* The code includes the legacy NetCDF c++ code (v. 4.2) but it only produces executables so it is not vendoring this library
* I am not familiar enough with the various executables produced in order to produce meaningful tests.  My sense is that they require relatively large datasets for proper use so CI testing might be a bit tricky.  Most also do not seem to have a `--help` flag, so I'm just making sure the expected executables are present for now.  Suggestions are very much appreciated!

File: recipes/tempest-extremes/meta.yaml (hunk 1 above)
Comment (chengzhuzhang) [reply recipes/tempest-extremes/meta.yaml:100]: Yes! I'm willing to be maintainer of this feedstock.

Comment (forsyth2) [reply recipes/tempest-extremes/meta.yaml:100]: Yes, I can be a maintainer.

**Comment by xylar** [comment]
@conda-forge/help-c-cpp, I know things must be busy.  A review would be appreciated when you are able.

File: recipes/tempest-extremes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}"
+
+  {% if mpi != 'nompi' %}
+  run_exports:
+    - {{ name }} * {{ mpi_prefix }}_*
+  {% endif %}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```
Comment (chrisburr) [inline recipes/tempest-extremes/meta.yaml:49]:
```suggestion
    - {{ compiler('cxx') }}
    - make
```

Comment (xylar) [reply recipes/tempest-extremes/meta.yaml:49]: Thanks @chrisburr, nice catch!

**Comment by xylar** [comment]
Thanks again, @chrisburr!

**Pull request merged**